src/backend/tcop/pquery.c

```diff
@@ -174,6 +177,9 @@ FreeQueryDesc(QueryDesc *qdesc)
 	UnregisterSnapshot(qdesc->snapshot);
 	UnregisterSnapshot(qdesc->crosscheck_snapshot);
 
+	if (qdesc->showstatctx)
+		cdbexplain_showStatCtxFree(qdesc->showstatctx);
+
 	/* Only the QueryDesc itself need be freed */
```

Comment:
Just a nothing-to-do remark.
This comment is not true, as we need to free `qdesc->showstatctx`. But as it came from postgres, which doesn't have such structure member, let's leave it as is.